packages/toolpad-app/cli/index.ts

```diff
@@ -43,6 +54,34 @@ async function runApp(cmd: 'dev' | 'start', { devMode = false, port }: RunComman
   cp.stdout?.pipe(process.stdout);
   cp.stderr?.pipe(process.stdout);
 
+  if (cp.stdout && cmd === 'dev' && !noBrowser) {
+    const checkedUrl = new URL(HEALTH_CHECK_URL, DEFAULT_URL);
```

Comment:
* The port is hardcoded in `HEALTH_CHECK_URL`. This would healthcheck the wrong server if Toolpad chooses a different port.
* I'd abstract this polling logic into its own `waitForAvailable(port: string): Promise<void>` function

Reply:
I've done these, and added some changes to use a `TOOLPAD_CLI_BROWSER` environment variable which supports either the names of browsers, or a "none" value to disable this behaviour.

The default browser that the script will attempt to open is Chrome.

Reply:
> The default browser that the script will attempt to open is Chrome.

Wouldn't `open` use the default browser of the user's operating system? If so, I believe it makes sense to use that as the default behavior